src/components/Cart.tsx

```diff
@@ -6,6 +6,14 @@ import { useProductState } from '../store/productState'
 export const Cart = () => {
   const cartItems = useCartStore((state) => state.getCartItems())
   const items = useProductState((state) => state.items)
+  const removeItemFromCart = useCartStore((state) => state.removeItemFromCart)
```

Comment:
just as an alternative you can also just destucture: 

const { cartItems, getCartItems, removeItemFromCart, clearCart .......} = useCartStore()

Reply:
good call, thanks man. will refactor accordingly 💪